src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,20 @@
+
+.dg_summary-list__item {
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  background-color: $gray-lightest;
```

Comment:
the background is not specific to an item so it can be removed.

Reply:
background removed